TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -422,6 +429,7 @@ public void DidChangeConfigurationParams(IEnumerable<string> arguments)
 
 #if EUROINFO_RULES
             typeCobolOptions.AutoRemarksEnable = TypeCobolConfiguration.AutoRemarks;
+            typeCobolOptions.UseCheckProgramName = UseCheckProgramName;
```

Comment:
The option can be used in a normal environment. The value UseCheckProgramName should be set in the constructor of TypeCobolOptions.
Use the value stored in TypeCobolConfiguration that is retrieved from the command line. If the configuration of the client change the new value won't be taken into account.